Source/Core/Core/CoreTiming.cpp

```diff
@@ -382,6 +387,12 @@ void CoreTimingManager::Throttle(const s64 target_cycle)
   }
 }
 
+void CoreTimingManager::ResetThrottle(s64 cycle)
+{
+  m_throttle_last_cycle = cycle;
```

Comment:
you should probably remove the cycle argument and make it reset to `GetTicks()`

Reply:
Good point. 

But that makes me wonder about what happens to `m_is_global_timer_sane` during savestates. Is there a bug there?

Reply:
I think the save-state code can only be triggered on the cpu thread from within an event, so it should always be true when the savestate code runs (and there would be more bugs if we were saving or loading with a non-sane global timer)